src/haddock/modules/refinement/openmm/__init__.py

```diff
@@ -0,0 +1,114 @@
+"""OpenMM refinement module for HADDOCK3."""
+import os
+import subprocess
+from contextlib import suppress
+from pathlib import Path
+
+from haddock.libs.libontology import PDBFile
+# from haddock.libs.libparallel import Scheduler
+from haddock.modules import BaseHaddockModule, get_engine
+
+
+# allow general testing when OpenMM is not installed
+with suppress(ImportError):
+    from haddock.modules.refinement.openmm.openmm import OPENMM
+
+
+RECIPE_PATH = Path(__file__).resolve().parent
+DEFAULT_CONFIG = Path(RECIPE_PATH, "defaults.yaml")
+
+
+def runSubprocess(command_to_run):

```

Comment:
I suggest moving this function to `libsubprocess` with the name `run_subprocess`.